### DISCUSSION TITLE
Getting error while building solution in Visual Studio 2019

### DISCUSSION THREAD

**Discussion by bkanzariya7** [opensourcecobol/opensourcecobol4j#465] [Q&A]
**Error**	LNK2019	unresolved external symbol GetTempPath2A referenced in function cobc_temp_name cobj

Can anyone please help me on this?

**Comment by yutaro-sakamoto** [comment]
@kio-watanabe 
Please investigate the problem.

**Comment by kio-watanabe** [comment]
@bkanzariya7 
We develop opensource COBOL 4J for Windows with Visual Studio 2022.
Could you try to build it using Visual Studio 2022?

Alternatively, you can download an executable file in [Release page](https://github.com/opensourcecobol/opensourcecobol4j/releases/tag/v1.1.2).

**Comment by kio-watanabe** [comment]
@bkanzariya7
We found that GetTempPath2A is not available by default on Windows 10.
Therefore, it can be built by changing GetTempPath2A in cobj.c to GetTempPathA.
The develop branch reflects this modification.  (Corrections: https://github.com/opensourcecobol/opensourcecobol4j/commit/f1c24724e2ad27edf41009c3d7a2eef246e123fa)

If you are using Windows 10, try the code in the develop branch.
